utils/profiles.ts

```diff
@@ -44,8 +44,8 @@ export function retrievedProfileToExtendedProfile(
     accepted_tos_version: profile.acceptedTosVersion,
     blocked_inbox_or_channels: profile.blockedInboxOrChannels,
     email: profile.email,
-    is_email_enabled: profile.isEmailEnabled === true,
-    is_email_validated: profile.isEmailValidated === true,
+    is_email_enabled: profile.isEmailEnabled !== false,
```

Comment:
why this change ?

Reply:
This property is `undefined` in old user profile (created before `isEmailEnabled` was available), but we still want to return an ExtendedProfile that has `is_email_enabled` set to `true` in this case.

Reply:
Same for is_email_validated.